content/contributors.md

```diff
@@ -7,22 +7,22 @@ The Gift of Open Source is open to everyone in our global community.
 Whether you’re a developer, student learning to code, event host, or just someone looking to have fun, you can help drive growth of open source and make positive contributions to the community.
 All backgrounds and skill levels are encouraged to participate.
 
-To get involved, contribute to any open source project hosted on GitHub.com, that has the `Gift-of-Open-Source` topic and contribute anything between December 1 and December 31 midnight (PST) in 2021. 
+To get involved, submit two pull requests to any open source project hosted on GitHub.com with the `Gift-of-Open-Source` topic, between December 1 and December 31 (11:59 PM PST) in 2021. 
 
 
 ## How to Contribute?
 
-Make X quality contributions to a participating repository. Here's how to get involved in 3 simple steps.
+Make atleast **2** quality contributions to a participating repository and you'll get a prize! Here's how to get involved in 3 simple steps:
 
-1. Register 
-1. Create a pull request in any participating repository. Make a contribution like improving existing implementations, fixing bugs, updating documentation, etc...
-1. Make X quality contributions. You can open a new issue mentioning the contribution you plan to make and open a pull request directly, and ask for approval from the repository maintainers.
+1. [Register to participate](https://pantheon.io/gift-open-source#register)
+1. [Create a pull request](https://opensource.com/article/19/7/create-pull-request-github) in any participating repository. The repo must have a `Gift-of-Open-Source` topic, and the issue must have a `gift-of-open-source` label or `giftofopensource` tag. Make a contribution like improving existing implementations, fixing bugs, updating documentation, etc...

```

Comment:
This should be an `or` instead of an `and`.
The repository should have the topic
OR
The issue should have the label.